crates/subspace-service/src/lib.rs

```diff
@@ -845,6 +858,55 @@ where
         SubspaceSyncOracle::new(config.force_authoring, sync_service.clone());
 
     if config.role.is_authority() || config.force_new_slot_notifications {
+        info!("subspace-service: PoT role = {:?}", config.pot_role);
+        let client_cl = client.clone();
+        let chain_info_fn: Arc<dyn Fn() -> Info<Block> + Send + Sync> =
+            Arc::new(move || client_cl.chain_info());
+        let pot_consensus = pot_components.consensus_state();
+        if config.pot_role.is_clock_master() {
+            let clock_master = ClockMaster::<Block, _, _>::new(
+                pot_components,
+                client.clone(),
+                sync_service.clone(),
+                network_service.clone(),
+                sync_service.clone(),
+                chain_info_fn,
+            );
+
+            let bootstrap_params = BootstrapParams::new(
+                client.chain_info().genesis_hash.to_fixed_bytes(),
+                Default::default(), // TODO:in initial key from cmd line or BTC
+                Slot::from_timestamp(
+                    *sp_timestamp::InherentDataProvider::from_system_time(),
+                    subspace_link.slot_duration(),
+                )
+                .into(),
+            );
+            task_manager.spawn_essential_handle().spawn_blocking(
+                "subspace-proof-of-time-clock-master",
+                Some("subspace-proof-of-time-clock-master"),
```

Comment:
I didn't mean to change `subspace-proof-of-time` to `pot`, even though it makes sense. `spawn_blocking` takes two arguments, second of which is called `group` and is meant to group multiple tasks corresponding to the same thing. So all tasks related to PoT should have `Some("pot")` rather than copying the name of the task.

Reply:
I'm fine either ways(as this group really has one task), changed as you suggested